robots.txt

```diff
@@ -0,0 +1,6 @@
+User-agent: *
```

Comment:
I think the default is to allow, so you only need the lines 5 and 6 for this behavior.